pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1577,6 +1577,10 @@ self: super: {
     ghc-paths = lsuper.ghc-paths.override { Cabal = null; };
   });
 
+  hledger_1_27_1 = super.hledger_1_27_1.overrideScope  (lself: lsuper: {
```

Comment:
Can you wrap this in a `doDistribute` call, so it is built by Hydra? Additionally, I'm fairly sure a plain `.override` would be enough which would be preferrable.

Reply:
fixed